src/common/__tests__/data/area.ts

```diff
@@ -16,3 +18,11 @@ export const rawAreaList = [
   { name: 'MSMBA' },
   { name: 'SEM' },
 ];
+
+/**
+ * An example of an [[Area]] entity
+ */
+export const appliedMath = Object.assign(new Area(), {
+  id: '4d4e622a-a843-43e1-8f53-c99532583178',
+  name: 'AM',
+} as Partial<Area>);
```

Comment:
Did a quick test on my end and it doesn't seem like this needs to be converted to `Partial<Area>`, I'd rather avoid unnecessary type conversions